src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/SoulboundRune.java

```diff
@@ -104,6 +102,21 @@ public ItemDropHandler getItemHandler() {
         };
     }
 
+    /**
+     * This method applies the {@link Soulbound} effect onto a given {@link ItemStack}.
+     * 
+     * @param item
+     *            The {@link ItemStack} to apply this effect to
+     */
+    public void apply(ItemStack item) {
```

Comment:
While this is being moved to a method, should add a persistent data tag for this rather than relying on the lore.

Reply:
Definitely agree with this, not my code, just refactored it.